quickwit/quickwit-doc-mapper/src/routing_expression/mod.rs

```diff
@@ -417,7 +416,7 @@ mod expression_dsl {
 
     fn identifier(input: &str) -> IResult<&str, &str> {
```

Comment:
can you add a comment with an example of what an identifier is...
same with key_identifier.